recipes/libcurl/all/conanfile.py

```diff
@@ -243,9 +245,23 @@ def build(self):
         else:
             autotools = Autotools(self)
             autotools.autoreconf()
+            # autoreconf is caalled with "--force" which regenerate all files.
+            # Because we want to use a patched config.sub for tvOS/watchOS, we
+            # need to call this patch after autoreconf.
+            self._patch_autoreconf()
```

Comment:
are you sure?

Reply:
I think so
```
~/repos/curl master ❯ ls -l config.{guess,sub}
-rwxr-xr-x  1 kleto.zan  staff  49348 Mar 30 17:54 config.guess
-rwxr-xr-x  1 kleto.zan  staff  35276 Mar 30 17:54 config.sub

~/repos/curl master ❯ autoreconf --install
glibtoolize: Remember to add 'LT_INIT' to configure.ac.

~/repos/curl master ❯ ls -l config.{guess,sub}
-rwxr-xr-x  1 kleto.zan  staff  49348 Mar 30 17:54 config.guess
-rwxr-xr-x  1 kleto.zan  staff  35276 Mar 30 17:54 config.sub

~/repos/curl master ❯ autoreconf --force --install
glibtoolize: putting auxiliary files in '.'.
glibtoolize: copying file './ltmain.sh'
glibtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
glibtoolize: copying file 'm4/libtool.m4'
glibtoolize: copying file 'm4/ltoptions.m4'
glibtoolize: copying file 'm4/ltsugar.m4'
glibtoolize: copying file 'm4/ltversion.m4'
glibtoolize: copying file 'm4/lt~obsolete.m4'
glibtoolize: Remember to add 'LT_INIT' to configure.ac.
configure.ac:123: installing './compile'
configure.ac:129: installing './missing'
docs/examples/Makefile.am: installing './depcomp'

~/repos/curl master ❯ ls -l config.{guess,sub}
-rwxr-xr-x  1 kleto.zan  staff  49348 Apr  3 14:44 config.guess
-rwxr-xr-x  1 kleto.zan  staff  35276 Apr  3 14:44 config.sub
```

Reply:
And if you are asking about autoreconf calling with force by conan, I think the code is here:

https://github.com/conan-io/conan/blob/f8e5abfddb87a01d31fe6d49590aa38a0d0837b4/conan/tools/gnu/autotoolstoolchain.py#L205

```
-------- Installing package libcurl/7.78.0 (16 of 16) --------
...
libcurl/7.78.0: Calling build()
libcurl/7.78.0: RUN: autoreconf --force --install
...
```